### ISSUE TITLE
command line arguments for 'cloud9-resize.sh' script

### ISSUE THREAD

**Issue by alexeyshishkin01** [aws-samples/aws-genai-llm-chatbot#446]
Dear all,

Is it possible to change 'cloud9-resize.sh' script in order to provide information about EBS volume number and its size as command line arguments ?

ex. running ./scripts/cloud9-resize.sh 1 200 would mean change the size of EBS volume '1' to '200' GB

An alternative can be to have two scripts - existing one 'cloud9-resize.sh' and a command line one, say 'cloud9-resize-batch.sh' which can be used with command line arguments

**Comment by alexeyshishkin01** [comment]
it looks like there's a ready-to-use solution - https://docs.aws.amazon.com/cloud9/latest/user-guide/move-environment.html#move-environment-resize

**Comment by massi-ang** [comment]
Hi Alex, can you explain what you are trying to do and why you would need the possibility to specify the EBS volume?

**Comment by alexeyshishkin01** [comment]
OK, let me rephrase my question:
I need to run ./scripts/cloud9-resize.sh in non-interactive way (inside of other script)
How to do it ?

**Comment by spugachev** [comment]
Now it's possible to run cloud9-resize.sh like this:
./cloud9-resize.sh 1 120
1 - volume id
2 - volume size

**Comment by alexeyshishkin01** [comment]
this is to confirm it works now

**Comment by alexeyshishkin01** [comment]
Could you pls. also update the corresponding documentation so that other users can run this script either interactively or non-interactively ?
